apps/functions-subscription/local.settings.json.example

```diff
@@ -6,9 +6,18 @@
     "COSMOSDB_ENDPOINT": "<put-here-the-the-cosmos-database-endpoint>",
     "COSMOSDB_DATABASE_NAME": "<put-here-the-name-of-cosmos-database>",
     "EVENTHUB_NAMESPACE": "<put-here-the-the-event-hub-namespace>",
-    "SUBSCRIPTION_REQUEST_EVENTHUB_NAME": "<put-here-the-name-of-the-event-hub>",
+    "SERVICEBUS_NAMESPACE": "<put-here-the-the-service-bus-namespace>",
+    "AzureWebJobsStorage": "<put-here-the-connection-string-to-a-storage>",
+
+    "LEASES_COSMOSDB_CONTAINER_NAME": "leases",
+    "SUBSCRIPTION_HISTORY_COSMOSDB_CONTAINER_NAME": "subscription-history",
+
     "SUBSCRIPTION_REQUEST_CONSUMER": "on",
+    "SUBSCRIPTION_REQUEST_EVENTHUB_NAME": "<put-here-the-name-of-the-event-hub>",
     "SubscriptionRequestEventHubConnection__fullyQualifiedNamespace": "<put-here-the-the-event-hub-namespace>",
-    "AzureWebJobsStorage": "<put-here-the-connection-string-to-a-storage>"
+
+    "EVENTS_PRODUCER": "on",
+    "EVENTS_SERVICEBUS_TOPIC_NAME": "<put-here-the-the-service-bus-topic-name>",
+    "EventProducerCosmosConnection__accountEndpoint": "<put-here-the-the-cosmos-database-account-endpoint>"
```

Comment:
**question**: should you add these environment variables to the terraform file of the function as well?

Reply:
Yes! Done ✅! Thanks!